src/handlers/notebook.ts

```diff
@@ -23,6 +24,22 @@ export class NotebookHandler implements IDisposable {
     this.debuggerService = options.debuggerService;
     this.notebookPanel = options.widget;
 
+    this.notebookPanel.toolbar.addItem(
```

Comment:
Wondering whether we could use the `insertAfter` method from the toolbar, so it would look something like this:

```typescript
this.notebookPanel.toolbar.insertAfter('kernelStatus', 'debugger-lifeCycle-button', new ToolbarButton(...));`
```

Reply:
Let's not do that for now, since the button is also added to other toolbars than the notebook.